packages/nocodb/src/lib/db/sql-data-mapper/lib/sql/sortV2.ts

```diff
@@ -29,6 +29,9 @@ export default async function sortV2(
     const column = await sort.getColumn();
     if (!column) continue;
     const model = await column.getModel();
+
+    const nulls = sort.direction === 'desc' ? 'LAST' : 'FIRST';
```

Comment:
what is the reason to name it `nulls`

Reply:
it's the same argument name used by knex and it's used to define the order of `null` values.

<img width="405" alt="Screenshot 2023-01-20 at 3 04 16 PM" src="https://user-images.githubusercontent.com/61551451/213662975-6c0637c3-a267-4435-aacd-5fd0d827a6d0.png">

Reply:
suggest to add a comment